Loop/Managers/DoseMath.swift

```diff
@@ -341,7 +342,7 @@ extension Collection where Element == GlucoseValue {
     ///   - maxBasalRate: The maximum allowed basal rate
     ///   - lastTempBasal: The previously set temp basal
     ///   - duration: The duration of the temporary basal
-    ///   - minimumProgrammableIncrementPerUnit: The smallest fraction of a unit supported in basal delivery
+    ///   - rateRounder: Closure that rounds recommendation to nearest supported rate. If nil, no rounding is performed
```

Comment:
[nit] The order of parameters here is different than in the actual function signature.